src/devices/Ws28xx/samples/Sk6812_Samples/Program.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
```

Comment:
I'd suggest to improve original sample and perhaps add a menu in there so you can choose which LED strip you have.

Also we should file an issue about renaming base class and changing namespace to something like `LedStrip` - do not do it in this PR though

Reply:
For this pull request I'd like to keep them separated. The next thing I'm trying to implement is support for WS2815, which currently don't seem to work although they're pretty similar to WS2812b. When I've got that working, I'd also rework the samples.

Reply:
fair enough